packages/node_modules/@webex/widget-message/src/container.js

```diff
@@ -181,7 +181,6 @@ export class MessageWidget extends Component {
 
     // To support changing destinations, we must reset the stores when mounting
     // so previous conversations to other destinations aren't displayed
-    this.props.resetConversation();
```

Comment:
This causes the `isFetching` status to reset, and is no longer needed in this location, as it's moved to `widget-space`